usecases/traverser/hybrid/searcher.go

```diff
@@ -228,8 +228,7 @@ func processDenseSearch(ctx context.Context,
 			return nil, err
 		}
 	} else {
-		targetVector := getTargetVector(params.TargetVectors)
-		vector = params.HybridSearch.NearVectorParams.VectorPerTarget[targetVector]
+		vector = params.HybridSearch.NearVectorParams.Vectors[0]
```

Comment:
Should we check the length here to ensure a panic can't occur?

Reply:
if we do not have a target vector, the targetvector is "", so we always have an entry. The whole thing is far from being ideal, but that is sadly used throughout the codebase